tests/generation/test_fsdp.py

```diff
@@ -46,10 +54,8 @@ def manage_process_group(func: Callable[..., Any]) -> Callable[..., Any]:
         """Manage the creation and destruction of the distributed process group for the wrapped function."""
 
         def wrapped(*args: Any, **kwargs: Any) -> Any:
-            if is_torch_mlu_available():
-                device_count = torch.mlu.device_count()
-            else:
-                device_count = torch.cuda.device_count()
+            torch_accelerator_module = getattr(torch, torch_device, torch.cuda)
+            device_count = torch_accelerator_module.device_count()
```

Comment:
Maybe it's better to use `backend_device_count` from `src/transformers/testing_utils.py`?

Reply:
@ydshieh , done, thx.